CMakeLists.txt

```diff
@@ -450,7 +544,7 @@ set(OFFICIAL_RELEASE_BUILD 1)
 # Default to generate a package appropriate for a distro; users override using command line with -D
 # -D writes variables into the cmake cache, and setting CACHE variables won't overwrite variables already in the CACHE
 if( "${DISTRO_ID}" MATCHES "ubuntu" )
-  set( CPACK_GENERATOR "DEB" CACHE STRING "cpack list: 7Z, DEB, IFW, NSIS, NSIS64, RPM, STGZ, TBZ2, TGZ, TXZ, TZ, ZIP" )
+  set( CPACK_GENERATOR "DEB" CACHE STRING "cpack list: 7Z, DEB, IFW, NSIS, NSIS64, RPM, STGZ, TBZ2, TGZ, TXZ, TZ, ZIP" FORCE )
 elseif ("${DISTRO_ID}" MATCHES "fedora" OR
         "${DISTRO_ID}" MATCHES "centos" OR
         "${DISTRO_ID}" MATCHES "redhatenterpriseserver" OR
```

Comment:
Missing "FORCE" for RPM and TGZ

Reply:
Maybe I don't understand the syntax here.  Isn't putting FORCE at the end of the string "" what we need here?

Reply:
I mean you need to add "FORCE" to line 458 and 461 as well

Reply:
ah ok.  I just pushed an update